app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -329,6 +327,27 @@ class CustomTransactionViewHolder(private val viewBinding: ItemTxTransferBinding
     }
 }
 
+class CreationTransactionViewHolder(private val viewBinding: ItemTxSettingsChangeBinding) :
+    BaseTransactionViewHolder<TransactionView.Creation>(viewBinding) {
+
+    override fun bind(viewTransfer: TransactionView.Creation, payloads: List<Any>) {
+        val resources = viewBinding.root.context.resources
+        val theme = viewBinding.root.context.theme
+        with(viewBinding) {
+            finalStatus.setText(viewTransfer.statusText)
+            finalStatus.setTextColor(ResourcesCompat.getColor(resources, viewTransfer.statusColorRes, theme))
+
+            dateTime.text = viewTransfer.dateTimeText
+            settingName.setText(viewTransfer.label)
+
+            // TODO: Add in future PR
+//            root.setOnClickListener {
+//                // TBD
```

Comment:
next time please be more specific - instead of just tbd something like navigate to tx details page would be more descriptive. No action needed

Reply:
At the time it wasn't clear whether we would need to fetch the creation tx from the backend or just show, what we get in the tx list. That's why there is a TBD. See https://github.com/gnosis/safe-android/issues/848 and https://github.com/gnosis/safe-android/issues/588